cirq/circuits/circuit.py

```diff
@@ -2302,6 +2296,30 @@ def _draw_moment_in_diagram(
         moment_groups.append((x0, max_x))
 
 
+def _get_global_phase_and_tags_for_op(op):
```

Comment:
Please provide type annotations for these functions.

Reply:
Done